input/fsh/codeSystem/codeSystem.fsh

```diff
@@ -0,0 +1,1319 @@
+Alias: $0203 = http://hl7.org/fhir/v2/0203
+Alias: $m49.htm = http://unstats.un.org/unsd/methods/m49/m49.htm
+
+CodeSystem: CIEL
```

Comment:
Overall comment - this CodeSystem realistically is a mixture of CIEL and custom concepts. It might be worthwhile to denote here that this isn't purely CIEL, maybe by calling it something like "CIEL (Kenya Extension)". Also note that this can have implications for long term CodeSystem management, since this CodeSystem FSH will probably have to be manually maintained over time.

Reply:
All the concepts in this section are from CIEL. I will also have a different CodeSystem for the rest of the concepts (Custom Concepts). Let me know if this approach makes sense.